lib/mysql.js

```diff
@@ -517,13 +520,13 @@ MySQL.prototype.buildExpression = function(columnName, operator, operatorValue,
     propertyDefinition) {
   if (operator === 'regexp') {
     if (operatorValue.ignoreCase)
-      console.warn('MySQL regex syntax does not respect the `i` flag');
+      g.warn('{{MySQL}} regex syntax does not respect the {{`i`}} flag');
```

Comment:
I dont think if regex needs to be translated either; I would say '{{MySQL regex}}'